packages/perseus/src/widgets/radio/choice-icon/cross-out-line.jsx

```diff
@@ -4,22 +4,19 @@
 import {StyleSheet, css} from "aphrodite";
 import * as React from "react";
 
-import {SAT_ICON_SIZE, LIBRARY_ICON_SIZE} from "./shared-styles.js";
+import {LIBRARY_ICON_SIZE} from "./shared-styles.js";
 
 // The "cross-out line" SVG is a bit bigger than the library icon, to provide
 // extra space for the cute rounded line cap.
 const CROSS_OUT_LINE_PADDING = 2;
 const CROSS_OUT_LINE_SIZE = LIBRARY_ICON_SIZE + CROSS_OUT_LINE_PADDING * 2;
-const SAT_CROSS_OUT_LINE_SIZE = SAT_ICON_SIZE + CROSS_OUT_LINE_PADDING * 2;
 
 /**
  * The "cross-out line" that appears over the icon when the choice has been
  * `crossedOut`.
  */
-function CrossOutLine(props: {color: string, sat?: boolean, ...}): React.Node {
-    const crossOutLineSize = props.sat
-        ? SAT_CROSS_OUT_LINE_SIZE
-        : CROSS_OUT_LINE_SIZE;
+function CrossOutLine(props: {color: string, ...}): React.Node {
+    const crossOutLineSize = CROSS_OUT_LINE_SIZE;
```

Comment:
nit: not a big deal, but I think it would be more consistent with the rest of the code in this file to just use the constant in the JSX.